implementations/tutorial_vqe_parallel.py

```diff
@@ -0,0 +1,208 @@
+# coding=utf-8
+r"""
+.. _vqe-parallel:
+
+VQE with parallel QPUs on Rigetti-Forest

```

Comment:
Not sure it has a hyphen!
```suggestion
VQE with parallel QPUs on Rigetti Forest
```